.circleci/config.yml

```diff
@@ -321,27 +315,22 @@ jobs:
             - 6f:4b:55:76:0e:cd:27:7d:ad:c3:28:38:53:69:5c:32
       - checkout
       - run:
-          name: Install semvertag
-          command: sudo pip install https://github.com/ministryofjustice/semvertag/archive/master.zip
+          name: Set environment
+          command: ~/project/.circleci/set_env.sh >> $BASH_ENV
       - run:
-          name: Bump version
-          command: semvertag bump patch << parameters.semver_stage >> >> ~/project/VERSION
+          name: Set version
+          command: |
+            export VERSION=${TF_WORKSPACE}-${CIRCLE_SHA1:0:7}
+
+            echo "export VERSION=${VERSION}" >> $BASH_ENV
+            echo "$VERSION" >> ~/project/VERSION
```

Comment:
A little clarification on what this is doing:
- Line 325 adds the environment variable to `$BASH_ENV` so it can be used by other steps in this job
- Line 326 persists the version to a file so it can be loaded by other jobs (through `set_env.sh`)